specification/2.0/README.md

```diff
@@ -3,13 +3,56 @@ Copyright 2014-2021 The Khronos Group Inc.
 SPDX-License-Identifier: CC-BY-4.0
 -->
 
-The glTF 2.0 Specification formerly hosted in this README.md has been moved
-to AsciiDoc markup. Published versions of the Specification are now located
-in the [glTF Registry](https://www.khronos.org/registry/glTF). The
-underlying AsciiDoc markup is in the `main` branch of the [glTF
-Specification Repository](https://www.github.com/KhronosGroup/glTF).
-
-A preview of the rendered output of the specification can be seen by
-opening the [`Specification.adoc`](Specification.adoc) file, but note that
-this preview does not include the properties reference and JSON schema
-reference, and will not render embedded mathematical equations correctly.
+![glTF 2.0](../figures/gltf.png)
+
+# The glTF 2.0 Specification
+
+The full specification is available in two forms:
+
+- [Specification as HTML](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html) - for viewing in a browser
+- [Specification as PDF](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.pdf) - for viewing as a document
+
+## Extensions
+
+Features not covered by the core glTF specification may be contributed by way of extensions. See the [glTF Extension Registry](https://github.com/KhronosGroup/glTF/tree/main/extensions) for details.
+
+## Contributing
+
+Although the file structure is finalized for glTF 2.0, error corrections and clarifications are most welcome here. Begin by checking the [list of issues](https://github.com/KhronosGroup/glTF/issues) to see if there has already been a discussion of the topic, and if not, file a new issue to get one started.
+
+After discussing the issue, if corrections are needed to the core glTF 2.0 specification, they may be submitted in the form of GitHub Pull Requests to this repository. The choice of file to edit depends on the section that needs editing:
+
+- **Section 5 (Properties Reference)** - This section is auto-generated using the [JSON Schema files](https://github.com/KhronosGroup/glTF/tree/main/specification/2.0/schema). Corrections to the wording must be made directly in those JSON files. Changes to the JSON structure are not being accepted at this time.
+
+- **Appendix A (JSON Schema Reference)** - This section is likewise auto-generated using the [JSON Schema files](https://github.com/KhronosGroup/glTF/tree/main/specification/2.0/schema).
+
+- **All other sections** - The remainder of the specification is contained in [Specification.adoc](https://github.com/KhronosGroup/glTF/blob/main/specification/2.0/Specification.adoc). General edits may be made here.
+
+## Building with GitHub Actions
+
+Opening a PR here will trigger a GitHub Action. Click on the "Actions" tab of the repository containing the edited branch, making careful note that this is likely to be your own fork of the glTF repository, not the main one. The most recent Action should display the name of your commit or PR.
```

Comment:
To avoid potential confusion with the branch name, "not the main one" -> "not the Khronos one".

Reply:
Done.